File: share-code/bai-tap-lap-trinh/python/comprehensive/72.py
days_english    = ["", "", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"]
days_vietnamese = ["", "", "Thứ hai", "Thứ ba", "Thứ tư", "Thứ năm", "Thứ sáu", "Thứ bảy", "Chủ nhật"]

your_number = 18

number_one = your_number // 10
number_two = your_number % 10

result = "Không giải mã được"

if number_one == 1 and 2 <= number_two <= 8:
    result = days_english[number_two]
elif number_one == 2 and 2 <= number_two <= 8:
    result = days_vietnamese[number_two]

print(result)

File: share-code/bai-tap-lap-trinh/cpp/09-tonghop/72.cpp
#include <iostream>
#include <string>
using namespace std;

int main()
{
    int number = 23;
    string result = "Không giải mã được";
    string dateVi[] = {"", "", "Thứ hai", "Thứ ba", "Thứ tư", "Thứ năm", "Thứ sáu", "Thứ bảy", "Chủ nhật"};
    string dateEn[] = {"", "", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"};
    int lang = number / 10;
    int day = number % 10;

    if (day >= 2 && day <= 8)
    {
        switch (lang)
        {
        case 1:
            result = dateEn[day];
            break;
        case 2:
            result = dateVi[day];
            break;
        }
    }

    cout << result << endl;

    return 0;
}